test/test_parabolic.jl

```diff
@@ -0,0 +1,59 @@
+module TestExamplesDGMulti2D
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+EXAMPLES_DIR = joinpath(examples_dir(), "dgmulti_2d")
+
+# Start with a clean environment: remove Trixi output directory if it exists
+outdir = "out"
+isdir(outdir) && rm(outdir, recursive=true)
+
+@testset "SemidiscretizationHyperbolicParabolic" begin
+
+  @trixi_testset "DGMulti 2D" begin
+
+    dg = DGMulti(polydeg = 2, element_type = Quad(), approximation_type = Polynomial(),
+                 surface_integral = SurfaceIntegralWeakForm(flux_central),
+                 volume_integral = VolumeIntegralWeakForm())
+    mesh = DGMultiMesh(dg, cells_per_dimension=(2, 2))
+
+    # test with polynomial initial condition x^2 * y
+    # test if we recover the exact second derivative
+    initial_condition = (x, t, equations) -> SVector(x[1]^2 * x[2])
+
+    equations = LinearScalarAdvectionEquation2D(1.0, 1.0)
+    equations_parabolic = Trixi.LaplaceDiffusion2D(1.0)
```

Comment:
```suggestion
    equations_parabolic = LaplaceDiffusion2D(1.0)
```
It's exported, isn't it?